packages/react-components/react-color-picker-preview/stories/src/ColorArea/ColorAreaDefault.stories.tsx

```diff
@@ -1,4 +1,33 @@
 import * as React from 'react';
+import { Button, makeStyles } from '@fluentui/react-components';
 import { ColorArea, ColorAreaProps } from '@fluentui/react-color-picker-preview';
 
-export const Default = (props: Partial<ColorAreaProps>) => <ColorArea {...props} />;
+const useStyles = makeStyles({
+  example: {
+    width: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+  previewColor: {
+    width: '50px',
+    height: '50px',
+    borderRadius: '4px',
+    border: '1px solid #ccc',
+  },
+});
+export const Default = () => {
+  const styles = useStyles();
+  const defaultColor = '#ff0000';
+  const [color, setColor] = React.useState(defaultColor);
+  const onChange: ColorAreaProps['onChange'] = (_, data) => setColor(data.color);
+  const resetSlider = () => setColor(defaultColor);
```

Comment:
![2024-10-22 11 00 22](https://github.com/user-attachments/assets/660abc36-80ad-4528-a8f9-71c460743afc)


Something is broken with controlled mode. Edit: looking at the implementation, it's not implemented

Reply:
Thanks for catching this